lib/platform/gitlab/index.ts

```diff
@@ -479,8 +479,9 @@ export async function getPr(iid: number): Promise<Pr> {
     const branch = (
       await gitlabApi.getJson<{ commit: { author_email: string } }>(branchUrl)
     ).body;
-    const branchCommitEmail =
-      branch && branch.commit ? branch.commit.author_email : null;
+    const branchCommitEmail = branch?.commit
+      ? branch.commit.author_email
+      : null;
```

Comment:
```suggestion
    const branchCommitEmail = branch?.commit?.author_email ?? null;
```

🤔 I think `branch` can't be `undefined` here. The `api.getJson` call would fail then.